sonic_y_cable/y_cable.py

```diff
@@ -1334,14 +1337,17 @@ def get_nic_voltage(physical_port):
     return voltage
 
 
-def upgrade_firmware(physical_port, fwfile):
-    """ This routine should facilitate complete firmware
-    upgrade of the Y cable on all the three ends of the
-    Y cable of the port specified.
-    All the components of the Y cable should be upgraded and committed
-    in their entirety by this single call subroutine.
-    This should return success code if firmware upgrade is successful
-    and an error code otherwise.
+def download_firmware(physical_port, fwfile):
+    """ This routine should basically download and store the firmware
+    of the Y cable on all the componenets of the Y cable of the port specified.
```

Comment:
Remove the first 'of the Y cable'; redundant.

Reply:
fixed